ferrischat_webserver/src/guilds/delete_guild.rs

```diff
@@ -1,6 +1,6 @@
-use rocket::http::Status;
-use rocket::response::{content, status};
+use actix_web::{web::Path, HttpResponse, Responder};
 
-pub async fn delete_guild(id: u64) -> status::Custom<&'static str> {
-    status::Custom(Status::Ok, "deleted guild test")
+/// DELETE /api/v1/guilds/{id}
+pub async fn delete_guild(Path(guild_id): Path<i64>) -> impl Responder {
+    HttpResponse::NoContent().body("deleted guild test")
```

Comment:
This defeats the purpose of a 204 `No Content`. Either remove the body, or send a 200 `OK`